eq-author/src/redux/auth/reducer.js

```diff
@@ -1,3 +1,4 @@
+import config from "../../config";
```

Comment:
You don't need to use relative imports
 ```suggestion
import config from "config";
```